Sources/Foundation/NSPathUtilities.swift

```diff
@@ -64,9 +64,8 @@ public func NSTemporaryDirectory() -> String {
         }
     }
 #if os(Android)
-    // Bionic uses /data/local/tmp/ as temporary directory. TMPDIR is rarely
-    // defined.
-    return "/data/local/tmp/"
+    // Fall back to /data/local/tmp/, which is used by Bionic, if TMPDIR is not defined.
+    return ProcessInfo.processInfo.environment["TMPDIR"] ?? "/data/local/tmp/"
```

Comment:
`TMPDIR` is already checked, see the 5-10 lines just above this Android block.

Reply:
You're absolutely right. I'll go ahead and close the PR.